pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/AndAuthorizer.java

```diff
@@ -0,0 +1,37 @@
+package org.pac4j.core.authorization.authorizer;
+
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.profile.CommonProfile;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+/**
+ * The conjunction of authorizers.
+ * @param <U> Type of profile
+ *
+ * @author Sergey Morgunov
+ * @since 3.4.0
+ */
+public class AndAuthorizer<U extends CommonProfile> implements Authorizer<U> {
```

Comment:
Can you add a test class for this `AndAuthorizer`? My level of test expectations is higher in pac4j-core...

Reply:
Yes, of course. 
Tests added.